apps/postgres-new/components/chat.tsx

```diff
@@ -114,7 +129,7 @@ export default function Chat() {
           {
             state: 'result',
             toolCallId: generateId(),
-            toolName: 'requestCsv',
+            toolName: 'requestSql',
```

Comment:
I transposed what was done for csv to sql so we can save the file on our end.

Reply:
This is probably the better approach anyway since it doesn't require sending the entire SQL file to the LLM (just like CSV). Assuming the SQL file they upload could be very large